### PR TITLE
add subcontractingTerms to Lots object

### PR DESCRIPTION
**Pull request by odscjen**
closes https://github.com/open-contracting/ocds-extensions/issues/185

### CODE REVIEW

**Comment by duncandewhurst** [comment]
Was this prompted by a support request?

Since the bill is still at an early stage, I'm wondering if it's better to wait until it is passed into law, at which point we can include a reference to it in the legal context section of the readme.

**Comment by odscjen** [comment]
It was prompted by https://crm.open-contracting.org/issues/8197 and agreed that we'd update the extension

File: extension.json [hunk 1]
```diff
@@ -14,6 +14,9 @@
   "schemas": [
     "release-schema.json"
   ],
+  "dependencies": [
+    "https://raw.githubusercontent.com/open-contracting-extensions/ocds_lots_extension/master/extension.json"
+  ],
```
Comment (jpmckinney) [inline extension.json:19]: I think we do not need this dependency.

File: README.md [hunk 2]
```diff
@@ -79,6 +79,10 @@ Report issues for this extension in the [ocds-extensions repository](https://git
 
 ## Changelog
 
+### 2022-07-18
+
+* Add `subcontractingTerms` field to `Lot` object
```
Comment (jpmckinney) [inline README.md:84]: Add period to be consistent with other entries

File: extension.json (hunk 1 above)
Comment (odscjen) [reply extension.json:19]: without this dependency the tests return AssertionErrors as Lot is undefined

Comment (jpmckinney) [reply extension.json:19]: Aha, put it under `testDependencies` not `dependencies`.

Comment (jpmckinney) [reply extension.json:19]: I've now added a couple bullets here: https://ocds-standard-development-handbook.readthedocs.io/en/latest/extensions/index.html

I didn't add this to https://github.com/open-contracting/standard_extension_template as `testDependencies` has no utility for third-party extensions.

Comment (odscjen) [reply extension.json:19]: ah! okay

**Review by jpmckinney — APPROVED** [review]

**Pull request merged**